tools/Microsoft.Health.Extensions.BuildTimeCodeGenerator/Sql/SqlVisitor.cs

```diff
@@ -22,6 +22,8 @@ namespace Microsoft.Health.Extensions.BuildTimeCodeGenerator.Sql;
 /// </summary>
 public abstract class SqlVisitor : TSqlFragmentVisitor
 {
+    private static readonly ISet<string> Keywords = new HashSet<string>() { "TableName" };
```

Comment:
TableName is not reserved, but in [Table](https://github.com/ccfromms/healthcare-shared-components/blob/main/src/Microsoft.Health.SqlServer/Features/Schema/Model/Table.cs), we have already defined a property named TableName. If I want to set a column named TableName in a certain sql table, it will throw an exception since TableName has been used in class Table.